src/bodies.cpp

```diff
@@ -1021,7 +1031,8 @@ void bodies::ConvexMesh::updateInternalData()
 
   shapes::Box box_shape(mesh_data_->box_size_.x(), mesh_data_->box_size_.y(), mesh_data_->box_size_.z());
   bounding_box_.setPoseDirty(pose);
-  bounding_box_.setPaddingDirty(padding_);
+  bounding_box_.setPaddingDirty(
+      padding_);  // the padding effect will most likely be smaller, but this is the worst case
```

Comment:
Please improve the comment (if it's really needed).
```suggestion
  // the padding effect will most likely be smaller, but this is the worst case
  bounding_box_.setPaddingDirty(padding_);  
```